src/renderer/dx/CustomTextLayout.cpp

```diff
@@ -434,6 +446,15 @@ CATCH_RETURN()
             }
         } while (tries < 2); // We'll give it two chances.
 
+        // Rearrange _glyphIndices
+        auto indicesDiff = textLength - actualGlyphCount;
+        while (indicesDiff > 0)
+        {
+            // Left shift the indices.
+            _glyphIndices.erase(_glyphIndices.begin() + glyphStart + actualGlyphCount);

```

Comment:
This is me trying to fix the issue mentioned above by left-shifting the entire array to avoid empty holes in `_glyphIndices`.